homework3/semi-uber/Server/data/requestLogs.json

```diff
@@ -1,12 +1 @@
,"time":1585167513117},{"url":"/","method":"POST","time":1585167580141},{"url":"/","method":"POST","time":1585221259385},{"url":"/","method":"POST","time":1585221274720},{"url":"/","method":"POST","time":1585222620976},{"url":"/","method":"POST","time":1585302804167},{"url":"/","method":"POST","time":1585302861073},{"url":"/:id","method":"GET","time":1585315283339},{"url":"/:id","method":"GET","time":1585315290846},{"url":"/:id","method":"GET","time":1585315317301},{"url":"/:id","method":"GET","time":1585315321502},{"url":"/:id","method":"GET","time":1585316063203},{"url":"/:id","method":"GET","time":1585316068967},{"url":"/:id","method":"GET","time":1585316072081},{"url":"/:id","method":"GET","time":1585316093673},{"url":"/:id","method":"GET","time":1585316138069},{"url":"/","method":"POST","time":1585316202995},{"url":"/:id","method":"GET","time":1585316203106},{"url":"/:id","method":"GET","time":1585316270482},{"url":"/:id","method":"GET","time":1585316289491},{"url":"/:id","method":"GET","time":1585316314639},{"url":"/:id","method":"GET","time":1585316347470},{"url":"/:id","method":"GET","time":1585316514876},{"url":"/:id","method":"GET","time":1585316615554},{"url":"/:id","method":"GET","time":1585316888444},{"url":"/:id","method":"GET","time":1585318719552},{"url":"/:id","method":"GET","time":1585319137211},{"url":"/:id","method":"GET","time":1585319150269},{"url":"/5","method":"GET","time":1585319546089},{"url":"/5","method":"GET","time":1585319613736},{"url":"/5","method":"GET","time":1585319621091},{"url":"/5e7e016b75584e4c8e398141","method":"GET","time":1585319623351},{"url":"/5e7e016b75584e4c8e398141","method":"GET","time":1585319625017},{"url":"/","method":"POST","time":1585319717321},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585319717438},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585319796540},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585319800870},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585319812118},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585319932142},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585320037579},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585320062428},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585320080808},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585320154781},{"url":"/5e7e0f2544c7f25725eb37b6","method":"GET","time":1585320165385},{"url":"/","method":"POST","time":1585320831569},{"url":"/5e7e137f3c6b125ba8899d31","method":"GET","time":1585320831695},{"url":"/5e7e137f3c6b125ba8899d32","method":"GET","time":1585320837703},{"url":"/5e7e137f3c6b125ba8899d32","method":"GET","time":1585320938415},{"url":"/5e7e137f3c6b125ba8899d32","method":"GET","time":1585320949403},{"url":"/5e7e137f3c6b125ba8899d32","method":"GET","time":1585320995491},{"url":"/5e7e137f3c6b125ba8899d32","method":"GET","time":1585321020007},{"url":"/5e7e137f3c6b125ba8899d31","method":"GET","time":1585321039792}]}
```

Comment:
I think that there is no need to send this changes to the repo. Let's go we just a blank file `requestLogs.json` that will be stored in th repo. But all the logs will be saved locally and you will not push them

Reply:
No need to keep blank file at repo. Program gonna create this file if is absent